docs/tutorials/index.md

```diff
@@ -3,8 +3,7 @@
 
 The following tutorials provide a hands-on introduction to using MLRun to implement a data science workflow and automate machine-learning operations (MLOps).
 
-- [**Quick-start Tutorial**](./01-mlrun-basics.html) 
-- [**Gen AI tutorials**](#gen-ai-tutorials)
+- [**Gen AI tutorials**](#gen--ai-tutorials)
```

Comment:
why? I see that in the latest documentation this links works  - https://docs.mlrun.org/en/latest/tutorials/index.html#gen-ai-tutorials